src/lib/containers/entity/annotations/CustomAdditionalPropertiesFieldTemplate.tsx

```diff
@@ -0,0 +1,68 @@
+import { FieldTemplateProps } from '@sage-bionetworks/rjsf-core'
+import React from 'react'
+import { FormControl, FormGroup, FormLabel } from 'react-bootstrap'
+
+/**
+ * FieldTemplate override for "additionalProperties" fields to customize the form appearance.
+ *
+ * Adapted from 'WrapIfAdditional' in @rsjf/core.
+ * @param props
+ * @returns
+ */
+export const CustomAdditionalPropertiesFieldTemplate = (
+  props: FieldTemplateProps & {
+    onKeyChange: (newKey: string) => void
+  },
+) => {
+  const {
+    id,
+    label,
+    children,
+    errors,
+    help,
+    description,
+    hidden,
+    required,
+    displayLabel,
+    classNames,
+    disabled,
+    onKeyChange,
+    readonly,
+  } = props
+
+  const keyLabel = `Key` // i18n ?
+  if (hidden) {
+    return <div className="hidden">{children}</div>
```

Comment:
reasoning for adding all the children to a hidden div, rather than returning an empty react fragment (for example)?

Reply:
It's just what the default implementation does: https://github.com/rjsf-team/react-jsonschema-form/blob/9626a092eec78c8c9c103865386b37ebf20727fe/packages/core/src/components/fields/SchemaField.js#L140